tests/test_expressions.py

```diff
@@ -791,6 +792,7 @@ def test_properties_from_dict(self):
 
     def test_convert(self):
         from collections import namedtuple
+        import pytz
```

Comment:
Does this need to be a dev. dependency now?

Reply:
Well, it gets pulled in by `pandas`. Is it better to make it explict?

Reply:
Ah I see. I guess it doesn't hurt being explicit? Not sure what's the best practice here, but I can imagine if for whatever reason pandas removes this dependency in the future, we'd break as a side-effect.

Reply:
Added